src/main/java/fruitshop/strategy/handler/PurchaseOperationHandler.java

```
@@ -0,0 +1,12 @@
+package fruitshop.strategy.handler;
+
+import fruitshop.db.Storage;
+import fruitshop.model.FruitTransaction;
+
+public class PurchaseOperationHandler implements OperationHandler {
+    @Override
+    public void operate(FruitTransaction transaction) {
+        int oldQuantity = Storage.fruits.get(transaction.getFruit());
+        Storage.fruits.put(transaction.getFruit(), oldQuantity - transaction.getQuantity());
```

what if there is not enough fruits?

lets throw an exception